group/templates/group/groupDetail.html

```diff
@@ -0,0 +1,519 @@
+          <p style="font-size:16px;color:gray">
+            {{ group_description }}
+            <a href="" class="btn btn-sm btn-default" data-toggle="modal" data-target="#myModal"
+               style="float:right">Edit</a>
+          </p>
+        </div>
+      </div>
+    </div>
+    {% endblock Group_intro_block %}
+  </div>
+  <hr> 
+  {% block contest_block %}
+  <div class="contest">
+    <p style="border-bottom:1px solid; padding:2px;  float:left; font-family:sans-serif">Contest</p>
+    <a href="" class="btn btn-sm btn-default" data-toggle="modal" data-target="#Addcontest"
+       style="float:right">Add contest</a>
+    <br>
+    <br>
+    <div class="panel-group" id="accordion">
+      <div class="panel panel-success">
+        <div class="panel-heading">
+          <h4 class="panel-title">
+            <a data-toggle="collapse" data-parent="#accordion" href="#collapseOne">Running-Contest</a>
+          </h4>
+        </div>
+        <div id="collapseOne" class="panel-collapse collapse in">
+          <div class="panel-body">
+            <table class="table table-hover">
+              <thead>
+                <tr>
+                  <th class='col-md-2'>Title</th>
+                  <th class='col-md-4'>Time</th>
+                </tr>
+              </thead>
+              <tbody>
+                {% for contest in c_data %}
+                <tr>
+                  <td>{{ contest.name }}</td>
+                  <td>{{ contest.time }}</td>
+                </tr>
+                {% endfor %}
+              </tbody>
+            </table>
+            <a href="" class="btn btn-sm btn-default" data-toggle="modal" data-target="#cViewall"
+               style="float:right">view all</a>
+          </div>
+        </div>
+      </div>
+      <div class="panel panel-default">
+        <div class="panel-heading">
+          <h4 class="panel-title">
+            <a data-toggle="collapse" data-parent="#accordion" href="#collapseTwo">Archive</a>
+          </h4>
+        </div>
+        <div id="collapseTwo" class="panel-collapse collapse">
+          <div class="panel-body">
+            <table class="table table-hover">
+              <thead>
+                <tr>
+                  <th class='col-md-2'>Title</th>
+                  <th class='col-md-4'>Time</th>
+                </tr>
+              </thead>
+              <tbody>
+                {% for archive in a_data %}
+                <tr>
+                  <td>{{ archive.name }}</td>
+                  <td>{{ archive.time }}</td>
+                </tr>
+                {% endfor %}
+              </tbody>
+            </table>
+            <a href="" class="btn btn-sm btn-default" data-toggle="modal" data-target="#aViewall"
+               style="float:right">view all</a>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+  {% endblock contest_block %}
+  <hr> 
+  {% block annowence_block %}
+  <ul id="volume_num">{{ vol }}</ul>
```

Comment:
The variable 'vol' may confuse django server because @drowsy810301 use this variable in base.html and you extend it.
Please use another name.
BTW, could you tell me what does it mean?